### PR TITLE
stETH on Optimism new contract with proposed mark

### PR DESCRIPTION
**Pull request by kovalgek**
Add new contract addresses
- https://www.notion.so/stETH-on-Mainnet-Optimism-deploy-plan-d03ff0382e584a49b16fe38ba34b76e7
- https://www.notion.so/stETH-on-Optimism-upgrade-announcement-and-action-plan-e7179cac8fa347959a4432e36cc0cd97

### CODE REVIEW

File: docs/deployed-contracts/index.md [hunk 1]
```diff
@@ -290,15 +290,24 @@ pagination_next: deployed-contracts/holesky
 
 #### Ethereum part
```
Comment (arwer13) [inline docs/deployed-contracts/index.md:291]: Missing LidoLocator new impl

File: docs/guides/verify-steth-on-optimism-upgrade-manual.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+The full list of the contracts in scope is provided in the Lido Multichain section for [Optimism](https://docs.lido.fi/deployed-contracts/#optimism).
+
+## Ethereum part
+
+### TokenRateNotifier
+
+:::info
+A standalone (non-proxy) contract proposed to plug to the protocol as a new [token rebase receiver](/contracts/lido-locator#posttokenrebasereceiver).
+
+Deployed address: [`0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823`](https://etherscan.io/address/0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823)
```
Comment (arwer13) [inline docs/guides/verify-steth-on-optimism-upgrade-manual.md:14]: I'm not sure duplicating the address here is better than refer to a single source of truth https://docs.lido.fi/deployed-contracts/#optimism

File: docs/guides/verify-steth-on-optimism-upgrade-manual.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+:::
+
+```bash
+# Account that is allowed to add or remove pushers.
+TOKEN_RATE_NOTIFIER_OWNER=0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c
```
Comment (arwer13) [inline docs/guides/verify-steth-on-optimism-upgrade-manual.md:19]: As I understand the document is intended for reading by those who wants to verify already deployed setup. So it is better to be structured not around env variables set upon deploying but around parameters observable in etherscan. Namely, arguments of constructors, initialialize functions and permissions setup.

File: docs/deployed-contracts/index.md [hunk 4]
```diff
@@ -7,6 +7,8 @@ pagination_next: deployed-contracts/holesky
 ## Core protocol
 
 - Lido Locator: [`0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb`](https://etherscan.io/address/0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb) (proxy)
+- Lido Locator: [`0x1D920cc5bACf7eE506a271a5259f2417CaDeCE1d`](https://etherscan.io/address/0x1D920cc5bACf7eE506a271a5259f2417CaDeCE1d) (impl)
```
Comment (arwer13) [inline docs/deployed-contracts/index.md:10]: The addresses are fine :like:

**Review by arwer13 — CHANGES_REQUESTED** [review]

File: docs/guides/verify-steth-on-optimism-upgrade-manual.md (hunk 3 above)
Comment (TheDZhon) [reply docs/guides/verify-steth-on-optimism-upgrade-manual.md:19]: Fair point! Will make another iteration

File: docs/deployed-contracts/index.md (hunk 4 above)
Comment (TheDZhon) [reply docs/deployed-contracts/index.md:10]: thanks!

File: docs/guides/verify-steth-on-optimism-upgrade-manual.md (hunk 2 above)
Comment (TheDZhon) [reply docs/guides/verify-steth-on-optimism-upgrade-manual.md:14]: I see your point, will probably remove them :+1:

File: docs/deployed-contracts/index.md [hunk 5]
```diff
@@ -290,15 +292,24 @@ pagination_next: deployed-contracts/holesky
 
 #### Ethereum part
 
+- (proposed) TokenRateNotifier: [`0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823`](https://etherscan.io/address/0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823)
+- (proposed) OpStackTokenRatePusher: [`0xd54c1c6413caac3477AC14b2a80D5398E3c32FfE`](https://etherscan.io/address/0xd54c1c6413caac3477AC14b2a80D5398E3c32FfE)
 - L1ERC20TokenBridge: [`0x76943C0D61395d8F2edF9060e1533529cAe05dE6`](https://etherscan.io/address/0x76943C0D61395d8F2edF9060e1533529cAe05dE6) (proxy)
```
Comment (loga4) [inline docs/deployed-contracts/index.md:297]: lets rename a proxy too (`L1ERC20TokenBridge` -> `L1LidoTokensBridge`

Comment (TheDZhon) [reply docs/deployed-contracts/index.md]: done

File: docs/guides/verify-steth-on-optimism-upgrade-manual.md (hunk 3 above)
Comment (TheDZhon) [reply docs/guides/verify-steth-on-optimism-upgrade-manual.md]: done

**Comment by kovalgek** [comment]
Approve!

**Review by arwer13 — APPROVED** [review]

**Review by TheDZhon — APPROVED** [review]
LGTM :guitar:

**Pull request merged**